realtime_traffic.rb

```diff
@@ -8,10 +8,17 @@ def initialize
   end
 
   def active_users
-    formatted_responses = {
-      active_users_30_minutes: response_hash[:rows][0][:metric_values][0][:value]
-    }
-    formatted_responses.to_json
+    begin
+      formatted_responses = {
+        active_users_30_minutes: response_hash[:rows][0][:metric_values][0][:value]
+      }
+    rescue StandardError => e
+      puts "#{e.message}"
+      formatted_responses = {active_users_30_minutes: "No data available"}
+      formatted_responses.to_json
```

Comment:
Could this be simpler maybe : `{ active_users_30_minutes: "No data available" }.to_json`?

Yeah, I see what you mean in the commit comment - it is odd you have to do this. Great spot though

Reply:
👍